src/actions/actions.js

```diff
@@ -0,0 +1,55 @@
+import { fetchNews } from '../api';
```

Comment:
You dont need directory `actions` with file `actions.js`, put everything into src/redux.js